ivadomed/keywords.py

```diff
@@ -8,6 +8,10 @@ class ConfigKW:
     MODEL_NAME = "model_name"
     MODIFIED_3D_UNET = "Modified3DUNet"
     DEBUGGING = "debugging"
+    EXPERIMENT_TRACKING = "experiment_tracking"
+    PROJECT_NAME = "project_name"
+    GROUP_NAME = "group_name"
+    RUN_NAME = "run_name"
```

Comment:
Thanks for the suggestion! I'm afraid it would be difficult to do so. There's a hierarchy that wandb follows (see figure attached) - experiments are categorized according to project name (sci-zurich) --> group name (SagT2w_training) --> runs. If we condense this, it would be diffcult for tracking the experiments efficiently (which is indeed the purpose of wandb). 

![wandb_crop](https://user-images.githubusercontent.com/53445351/153121974-0148fe57-8bfa-435d-a278-3ea33efec449.PNG)

That said, I'll read up more on this and see if they can be shortened in any way possible!

Reply:
@naga-karthik I think Uzay mean the keyword in ConfigKW. 

If you create a new keyword class e.g. `WandBConfigKW` (which essentially indicates a new context) this can potentially allow people to know that `WandBConfigKW.GROUP` can only mean group in the WandB context and allow to you to keep it shorter while keeping the code more readable as it will appear int eh code as "WandBConfigKW.GROUP` which hints the context pretty explicitly. 

Also I think EXP_TRACKING is probably clearly enough. This is also the advantage of using keywords class: easy refactoring/hinting to developers without changing the underlying key string.

Reply:
Thank you for the clarification @dyt811! I have created a new keyword class as suggested (see 265f8d28c75eef5234dfcb989f49c5acf1ec7b2c for details). 

Also, I have removed the variable EXP_TRACKING because I have observed that currently ivadomed logs the loss curves on tensorboard by default and the images are logged only when `debugging=true`. To follow this, I did the same with wandb as well (log losses/metrics by default and images when debugging=true).